### DISCUSSION TITLE
Mason just did not udpate the lsp server.

### DISCUSSION THREAD

**Discussion by 4t8dd** [LazyVim/LazyVim#4234] [Extra Requests]
Is this OK?
I found this because my rust-analyzer is so old and nvim just reach it from mason.
At last I found the version in mason is 2023 version.
I never pay attention to this. I jsut update from lazyvim, I think lazy should control this.

**Comment by dpetka2001** [comment]
LazyVim has no control over this. Mason ships whatever version they have available. If a LSP server version should be updated, that's Mason's responsibility.
